packages/cloudflare/package.json

```diff
@@ -24,6 +24,11 @@
       "types": "./dist/api/index.d.ts",
       "default": "./dist/api/index.js"
     },
+    "./config": {
```

Comment:
is this needed given l32?
should this be re-exported from `api/index.ts` (by name)?

Reply:
what about also re-exporting cache stuff from the index? 
/cc @james-elicx

Reply:
> is this needed given l32?

no, my bad, thanks 👍

Reply:
> should this be re-exported from api/index.ts (by name)?

done

Reply:
> what about also re-exporting cache stuff from the index?

My general preference is to have explicit imports for optional features in a public API, instead of lumping it in a barrel file. In this case, defineCloudflareConfig should always be used, but the cache adapters are all entirely optional + independent, and may/may not be used depending on use case.

I don't particularly mind either way, but that's my 2c.

Reply:
ATM we are in this weird state where `index.ts` re-export some of the public API but not all - it can be confusing.

Any drawback re-exporting from the barrel file? If the files can be tree shaken, I would prefer that.

Reply:
d1TagCache, kvIncrementalCache, memoryQueue ?

Reply:
done

Reply:
oof this is annoying.... exporting them from the index file doesn't work well because this makes `next build` also import them when it `require`s the Next.js config file (since we import `setupOpenNextCloudflareForDev` there) and the problem is that Next's module resolution doesn't resolve all the necessary files 😕 

![Screenshot 2025-02-26 at 14 50 04](https://github.com/user-attachments/assets/7856b7eb-dc6e-4d2f-ada9-a89846af04d2)

So this is actually no working, potentially changes upstream in aws would be needed to enable this 😕

Reply:
@vicb I'm going to revert the change and keep the utils importable from the other files unless you have some suggestion?

Reply:
I guess revert the change for now and figure out later...
Looks like there is a mix of import with and w/o extensions